operations/kubernetes/encryption/README.md

```diff
@@ -136,6 +147,10 @@ Encryption status is obtained through the `--status` switch of the `encryption.s
 
     ```text
     k8s encryption status
+    changed: 1970-01-01 00:20:00+0000
+    ncn-m001: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907 identity
+    ncn-m002: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907 identity
+    ncn-m003: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907 identity
     current: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907
     goal: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907
     etcd: aescbc-46d5bd8c2001d07ded05687fe51b517033dc609e69fe4dddaa6e05656cf6e907
```

Comment:
Could add at the end of "This indicates that all secret data in etcd is now encrypted with this key provider's name."
Note the return code for status is zero in this case.